index.html

```diff
@@ -4,5 +4,9 @@
     overflow: hidden;
   }
 </style>
-<canvas />
+<div style="width: 100%; height: 100%; background-color: #0e0e0e">
```

Comment:
You should be able to get this for free if you apply it on the body or html element via the style tag we have if you want.